### PR TITLE
Updated login and onboarding layout

### PR DESCRIPTION
**Pull request by tmafaralala**
# Refactored layout for login and onboarding to work in both landscape and portrait.

- # Onboarding Potrait & Landscape
Screenshot 2022-03-15 at 18 05 54  Screenshot 2022-03-15 at 18 06 19

- # LogIn Potrait & Landscape
Screenshot 2022-03-15 at 18 06 38Screenshot 2022-03-15 at 18 07 01

### CODE REVIEW

**Review by smabhena — DISMISSED** [review]

File: free-tunes/Controllers/LoginViewController.swift [hunk 1]
```diff
@@ -64,44 +64,4 @@ class LoginViewController: UIViewController {
         passwordInput.emptyFieldError()
     }
 
-<<<<<<< HEAD
-=======
```
Comment (bgouws) [inline free-tunes/Controllers/LoginViewController.swift:68]: PR looks good, just make sure that this is intentional 😄 I see here you removed both `HEAD` and `develop` when you were solving merge conflicts so maybe just make sure this branch is working as intended before merging.

**Review by bgouws — DISMISSED** [review]

**Comment by bgouws** [comment]
Layouts are looking great! 🥳

File: free-tunes/Controllers/LoginViewController.swift (hunk 1 above)
Comment (tmafaralala) [reply free-tunes/Controllers/LoginViewController.swift:68]: I missed that😄, it works perfectly now

**Review by smabhena — APPROVED** [review]
👍

**Review by simekanidev — APPROVED** [review]

**Review by kieranwoodrow — APPROVED** [review]

**Pull request merged**